test/controllers/organizations/home_controller_test.rb

```diff
@@ -4,6 +4,7 @@ class Organizations::HomeControllerTest < ActionDispatch::IntegrationTest
   setup do
     @organization = create(:organization, :with_page_text)
     @organization.save
+    @pets = create_list(:pet, 4, :with_image, organization_id: @organization.id)
```

Comment:
Should we need this? The Index action should be performing a query to retrieve 4 pets from the given Org, and we should be able to access that via the instance var within a controller test?